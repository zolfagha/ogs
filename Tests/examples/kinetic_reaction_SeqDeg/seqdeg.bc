#BOUNDARY_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POLYLINE RIGHT_BOUNDARY
 $DIS_TYPE
  CONSTANT 0.0
#BOUNDARY_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POLYLINE LEFT_BOUNDARY
 $DIS_TYPE
  CONSTANT 4.0
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  SO42-
 $GEO_TYPE
  POLYLINE LEFT_BOUNDARY
 $DIS_TYPE
  CONSTANT  1.0
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  O2
 $GEO_TYPE
  POLYLINE LEFT_BOUNDARY
 $DIS_TYPE
  CONSTANT  0.26
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Tracer
 $GEO_TYPE
  POLYLINE INJECTION_LINE
 $DIS_TYPE
  CONSTANT  1.0
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  CH2O
 $GEO_TYPE
  POLYLINE INJECTION_LINE
 $DIS_TYPE
  CONSTANT  1.0
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Tracer
 $GEO_TYPE
  POLYLINE NOTINJECTION_LINE
 $DIS_TYPE
  CONSTANT  1.0e-20
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  CH2O
 $GEO_TYPE
  POLYLINE NOTINJECTION_LINE
 $DIS_TYPE
  CONSTANT  1.0e-20
#STOP
