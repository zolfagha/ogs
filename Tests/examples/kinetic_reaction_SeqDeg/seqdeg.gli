GeoSys-BC: Boundary Conditions ------------------------------------------------
#POINTS
0  0.0    0.0 0.0 0.0 0.0 0.0 0.0
1  0.0    5.0 0.0 0.0 0.0 0.0 0.0
2  400.0  0.0 0.0 0.0 0.0 0.0 0.0
3  400.0  5.0 0.0 0.0 0.0 0.0 0.0
4  0.0    3.0 0.0 0.0 0.0 0.0 0.0
5  0.0    2.9 0.0 0.0 0.0 0.0 0.0
#POLYLINE
 $NAME 
  LEFT_BOUNDARY
 $EPSILON
  1.0e-4
 $POINTS
  0
  1
#POLYLINE
 $NAME 
  RIGHT_BOUNDARY
 $EPSILON
  1.0e-4
 $POINTS
  2
  3
#POLYLINE
 $NAME 
  INJECTION_LINE
 $EPSILON
  1.0e-4
 $POINTS
  4
  1
#POLYLINE
 $NAME 
  NOTINJECTION_LINE
 $EPSILON
  1.0e-4
 $POINTS
  0
  5 
#POLYLINE
 $NAME 
  X_LINE
 $EPSILON
  1.0e-4
 $POINTS
  1
  3
#POLYLINE
 $NAME 
  Y_LINE
 $EPSILON
  1.0e-4
 $POINTS
  0
  1
#POLYLINE
 $NAME 
  BOUND
 $EPSILON
  1.0e-4
 $POINTS
  0
  2
  3
  1
  0
#STOP
