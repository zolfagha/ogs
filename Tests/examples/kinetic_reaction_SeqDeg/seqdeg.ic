#INITIAL_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 0.0
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Tracer
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0e-20
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  CH2O
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0e-20
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  SO42-
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0e-20
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  O2
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0e-20
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  CO2
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0e-20
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  HCO3-
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0e-20
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  H2S
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0e-20
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  H2O
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Biomass_oxygen
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Biomass_sulfate
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0
#STOP
