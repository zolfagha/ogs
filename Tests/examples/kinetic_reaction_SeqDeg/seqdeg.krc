GeoSys Kinetic Reactions
#KINREACTIONDATA      
 $SOLVER_TYPE
	1                ; Solver type
 $RELATIVE_ERROR	
	1.e-6		 ; rel. Error
 $MIN_TIMESTEP	
	1.e-6		; min Timestep
 $INITIAL_TIMESTEP	
	1.e-4       ; init Timestep
 $BACTERIACAPACITY
  1.0
 $NO_REACTIONS
   POLYLINE PLY_BC_LEFT

#REACTION ; 
 $NAME
   Oxygen_Reduction ;
 $TYPE
   monod;
 $BACTERIANAME
   Biomass_oxygen
 $EQUATION
  1 CH2O + 1 O2 = 1 CO2 + 1 H2O; 
 $RATECONSTANT
 2.0e-4  1.0   ; max growth rate my_max , exponent
 $GROWTH
    1
 $MONODTERMS
  ;2   ; Number of monod terms in kinetic equation
  CH2O   0.06      1.0 ; for each monod term: species concentration order
  O2     3.125e-4  1.0
 
#REACTION ; 
 $NAME
   Sulfate_Reduction ;
 $TYPE
   monod;
 $BACTERIANAME
   Biomass_sulfate
 $EQUATION
  2 CH2O + 1 SO42- = 2 HCO3- + 1 H2S;
 $RATECONSTANT
 5.0e-5  1.0   ; max growth rate my_max , exponent
 $GROWTH
    1
 $MONODTERMS
  ;2   ; Number of monod terms in kinetic equation
  CH2O    0.06  1.0 ; for each monod term: species concentration order
  SO42-   0.13  1.0
 $INHIBITIONTERMS
  ;1
  O2    2.0e-5  1.0
#STOP





 $DECAYTERM
  ; 
 $INHIBITIONTERMS
  ;0
 $PRODUCTIONTERMS
  ;0
  CH2O     -2.0
  SO42-    -2.0
  HCO3-    +2.0
  H2S      +1.0
