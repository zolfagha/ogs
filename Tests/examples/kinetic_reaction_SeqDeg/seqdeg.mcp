#COMPONENT_PROPERTIES  ; comp1
 $NAME
  Tracer ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#COMPONENT_PROPERTIES  ; comp2
 $NAME
  CH2O ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#COMPONENT_PROPERTIES  ; comp3
 $NAME
  SO42- ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#COMPONENT_PROPERTIES  ; comp4
 $NAME
  O2 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#COMPONENT_PROPERTIES  ; comp5
 $NAME
  CO2 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#COMPONENT_PROPERTIES  ; comp6
 $NAME
  HCO3- ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#COMPONENT_PROPERTIES  ; comp7
 $NAME
  H2S ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#COMPONENT_PROPERTIES  ; comp8
 $NAME
  H2O ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#COMPONENT_PROPERTIES  ; comp9
 $NAME
  Biomass_oxygen ;
 $MOBILE
  0; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#COMPONENT_PROPERTIES  ; comp10
 $NAME
  Biomass_sulfate ;
 $MOBILE
  0; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#STOP
