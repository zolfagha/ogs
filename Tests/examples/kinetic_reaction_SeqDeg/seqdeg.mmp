GeoSys-MMP  Material Medium Properties ------------------------------------
#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  2
 $GEOMETRY_AREA
  1.0
 $POROSITY
  1  0.3
 $TORTUOSITY
  1   1.000000e+000
 $PERMEABILITY_TENSOR
  ISOTROPIC  0.41666
 $MASS_DISPERSION
  1  1.0  0.001
 $DENSITY
  1  1500.0
 #STOP
