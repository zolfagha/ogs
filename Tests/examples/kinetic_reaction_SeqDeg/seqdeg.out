#OUTPUT // profile
 $NOD_VALUES
  HEAD
  Tracer
  CH2O
  SO42-
  O2
  CO2
  HCO3-
  H2S
  Biomass_oxygen
  Biomass_sulfate
  eta_mob_0
  eta_mob_1
  eta_mob_2
  eta_mob_3
  eta_mob_4
  eta_mob_5
  xi_mob_0
  xi_mob_1
  xi_mob_rate_0
  xi_mob_rate_1
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 10
#STOP




#OUTPUT // profile
 $NOD_VALUES
  HEAD
  tracer
  c12h1
  c13h1
  c12h2
  so4
  o2
 $GEO_TYPE
  POLYLINE X_LINE
 $DAT_TYPE
  TECPLOT
 $TIM_TYPE
  STEPS 10
#OUTPUT // profile
 $NOD_VALUES
  HEAD
  tracer
  c12h1
  c13h1
  c12h2
  so4
  o2
 $GEO_TYPE
  POLYLINE Y_LINE
 $DAT_TYPE
  TECPLOT
 $TIM_TYPE
  STEPS 10
#STOP
